lib/components/molecules/Typeahead/Typeahead.story.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './Typeahead.mock';
+
+// Import Styled Component to showcase variations
+import Typeahead, { TypeaheadVanilla } from '.';
+import ListItem from './ListItem';
```

Comment:
You should not be required to import it separately,  hint <Typeahead.ListItem />